lib/components/narrative/narrative-routing-results.js

```diff
@@ -32,12 +31,8 @@ class NarrativeRoutingResults extends Component {
     if (mainPanelContent) return null
 
     return (
-      routingType === 'ITINERARY'
-        ? <TabbedItineraries itineraryClass={itineraryClass} itineraryFooter={itineraryFooter} itineraries={itineraries} customIcons={customIcons} />
-        : <NarrativeProfileOptions
-          itineraryClass={itineraryClass}
-          customIcons={customIcons}
-        />
+      routingType === 'ITINERARY' &&
```

Comment:
I'm not sure this check for routingType is necessary. Although... maybe we should keep it in place because we may have a new routing type (`MATRIX`) in short order...